qutebrowser/config/configutils.py

```diff
@@ -60,35 +63,48 @@ class ScopedValue:
 
     value = attr.ib()  # type: typing.Any
     pattern = attr.ib()  # type: typing.Optional[urlmatch.UrlPattern]
+    # An increasing counter of order this value was inserted in.
+    insert_id = attr.ib(cmp=False)  # type: int
```

Comment:
What's the point of setting `cmp=False` here if you set `cmp=False` for the whole `ScopedValue` class anyways?